engine/settings/dev.py

```diff
@@ -67,3 +68,12 @@
     EXTRA_MESSAGING_BACKENDS = [("apps.base.tests.messaging_backend.TestOnlyBackend", 42)]
     TELEGRAM_TOKEN = "0000000000:XXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXXXXX"
     TWILIO_AUTH_TOKEN = "twilio_auth_token"
+
+INTERNAL_IPS = [
+    "127.0.0.1",
+]
+
+# the below two lines make it possible to use django-debug-toolbar inside of docker locally
+# https://knasmueller.net/fix-djangos-debug-toolbar-not-showing-inside-docker
+hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
+INTERNAL_IPS += [".".join(ip.split(".")[:-1] + ["1"]) for ip in ips]
```

Comment:
this allows the `django-debug-toolbar` to work when running the backend inside of `docker-compose`